base_synchro/wizard/__init__.py

```diff
@@ -1,3 +1,3 @@
-# -*- coding: utf-8 -*-
+#  coding: utf-8
```

Comment:
remove the coding line from all py files.

Reply:
done